src/esp/gfx/WindowlessContext.cpp

```diff
@@ -62,120 +62,33 @@ const int MAX_DEVICES = 128;
 
```

Comment:
Hmm, I think the `CHECK_EGL_ERROR`, `MAX_DEVICES`, the `<fcntl.h>` includes and the `// Based on code from:` comments aren't needed anymore either :)